ehrql/debugger.py

```diff
@@ -37,7 +45,12 @@ def debug(
     """
     line_no = inspect.getframeinfo(sys._getframe(1))[1]
     elements = [element, *other_elements]
-    element_reprs = [repr(el) for el in elements]
+
+    if hasattr(element, "__repr_related__") and elements_are_related_series(elements):
+        element_reprs = [element.__repr_related__(*other_elements)]
```

Comment:
It would be good to separate out elements that don't have a `"__repr__related__"` method before this; then e.g. `debug("foo", patients.date_of_birth, patients.date_of_death`)` can still put together the patient table.

Reply:
Yeah, I originally suggested something similar but Peter liked the simplicity of the all-or-nothing approach. I think we can safely _expand_ the range of circumstances in which we display related series later without worrying about backwards compatibility. So I guess it makes sense to start small for now.